_pages/channels/truebeginners.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: channel
+title: TrueBeginners
+channel: truebeginners
+permalink: /channels/truebeginners/
+parent: channels
+---
+
+A "No Judgement Zone" for newbies, a good place to get your feet wet and ask all kinds of questions as to where to start, or for clarification on things that may be confusing.
+
+This channel is *usually* for less language-specific questions (eg: learning about classes vs. learning about C#), but feel free to ask here anyway.
+
+### Links to various getting started documents
```

Comment:
Let’s add at least one link:
- [Free eBooks for beginning developers](https://www.packtpub.com/packt/offers/free-learning)

Alternatively, let’s remove/comment out the title.